cg/store/filters/status_order_filters.py

```diff
@@ -28,6 +28,19 @@ def filter_orders_by_ticket_id(orders: Query, ticket_id: int | None, **kwargs) -
     return orders.filter(Order.ticket_id == ticket_id) if ticket_id else orders
 
 
+def filter_orders_by_search(orders: Query, search: str | None, **kwargs) -> Query:
+    if search:
+        orders = orders.join(Order.customer)
+        orders.filter(
+            or_(
+                Order.id.startswith(search),
```

Comment:
Should this not be equals? Or would we want to search for 967 and get all 9670, 9671 etc back?